### PR TITLE
nolo tlc ciel 13_6

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Ciel/Day 13/13_01_02_CIEL13_6.txt [hunk 1]
```diff
@@ -186,27 +186,27 @@ Noel holds a single deadly weapon in her hand.
...
 -- Page 117, Offset 35593.
 -- 大事なのは、生意気にもわたしをどうにかできると思ってるその目つきをどう教育するか、なんだし」
-The real question is how I should fix that look of yours that arrogantly thinks you can do something about me."
+The real question is... how do I wipe that confident look right off your face?"
 }
 [sha:a407b30c017cc3204b6419950704232c8c85fc19]{
 -- Page 118, Offset 35594.
 -- ……来る。武器の有利不利、身体能力の差はどうあれ、受けるのは二回が限界だ。
-...She's coming. Regardless of her weapon's strong and weak points, or our differences in physical strength, I can only take two of those at max.
+...She's coming. Regardless of the disadvantages and advantages of our respective weapons, and the difference in strength, the fact of the matter is that I can only take two hits at most.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_6.txt:199]: or the difference in our strengths

File: script/Ciel/Day 13/13_01_02_CIEL13_6.txt [hunk 2]
```diff
@@ -216,27 +216,27 @@ Without straining or relaxing my attention, I sweep backward as if I were fallin
 [sha:096dfb8e006e7e3396668c4a97c901d9c5a9adc9]{
 -- Page 120, Offset 35598.
 -- ブリッジに近い姿勢で、飛びこんできたノエルごと凶器をやり過ごす。
-With a bridge-like stance, I let Noel fly past together with her deadly weapon.
+Assuming a posture akin to a bridge, I let Noel slip right over me.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_6.txt:219]: what does this even mean, what

File: script/Ciel/Day 13/13_01_02_CIEL13_6.txt [hunk 3]
```diff
@@ -246,7 +246,7 @@ From a position below, I decide to aim for that line...
 [sha:327b47a1c09e89600cea16ee1afe1937023e4a4b]{
 -- Page 122, Offset 35604.
 -- 串を持ったノエルの肘を躱しきれず、次の瞬間には無様に弾き飛ばされていた。
-Unable to avoid the elbow of the arm holding the skewer, I immediately get sent awkwardly flying away.
+Unable to avoid the elbow of the arm holding the skewer, I'm pathetically blown away the next thing I know.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_6.txt:249]: skewer, the next thing I know, I'm

File: script/Ciel/Day 13/13_01_02_CIEL13_6.txt [hunk 4]
```diff
@@ -256,37 +256,38 @@ Unable to avoid the elbow of the arm holding the skewer, I immediately get sent
 [sha:84e29cd556e81579581e13389de7880fae3f9709]{
 -- Page 123, Offset 35606.
 -- 背中から地面に落ち、何度か回転して勢いを流す。
-I fall down on my back before rolling around several times to break the momentum.
+I fall on my back before rolling a couple of times to break the momentum.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_6.txt:259]: my momentum

File: script/Ciel/Day 13/13_01_02_CIEL13_6.txt [hunk 5]
```diff
@@ -296,77 +297,78 @@ On the other hand, it seems that the enemy is in a similar position.
...
 -- 死徒とは“生命のバージョンアップ”だ。
-One could argue that a Dead Apostle is an 'upgraded version of a life'.
+One could argue that a Dead Apostle is an 'upgraded lifeform'.
 }
 [sha:4146e19067238612f024c6db8a7391721bfaa114]{
 -- Page 132, Offset 35626.
 -- 吸血鬼になる事でその身体の最適解ともいえる美貌を得る事もあるし、人間以上の快楽を与える名器にもなり得る。
-By turning into a vampire, you can achieve the highest level of beauty for that body, and you could also become a masterpiece that offers more pleasure than any other human being.
+// nasu really had to write about hitting the vampussy didnt he
+Upon becoming a vampire, one's body morphs into the most beautiful form it can possibly take, creating a masterpiece that is more pleasurable than any human being.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_6.txt:361]: this is one of those lines i can't wait to see posted everywhere and scrutinized. gonna be fun!

File: script/Ciel/Day 13/13_01_02_CIEL13_6.txt [hunk 6]
```diff
@@ -461,12 +463,13 @@ I mean, I am a vampire after all! Thinking about consequences and holding back i
 [sha:a221a0d351b6944bbfc82b20c434faf6c9e506d2]{
 -- Page 141, Offset 35647.
 -- 「馬鹿じゃないの？　アレは生き返るだけで死なない訳じゃないでしょ。殺せばちゃんと死ぬのよ、あの人は」
-"Are you stupid? Just because she can revive doesn't mean she cannot die. A person like her will die properly if you kill her."
+// ahhhh shit
+"Are you stupid? The fact that she comes back to life doesn't mean that she can't die. Just like anyone else, she'll die if you kill her."
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_6.txt:467]: i spoke too soon

File: script/Ciel/Day 13/13_01_02_CIEL13_6.txt [hunk 7]
```diff
@@ -461,12 +463,13 @@ I mean, I am a vampire after all! Thinking about consequences and holding back i
...
 }
 [sha:c0257ad9efedf06fbe5f563183e1df32197fa262]{
 -- Page 142, Offset 35648.
 -- 「だから、ね？　ずっと殺し続けておけばいい。たとえば濃硫酸のプールに浸しておくとかどう？　生き返っても即死する状況にしておけば、あの子は永遠に生き返るけど“生きていない”事になるでしょう？」
-"So here's my plan. I'll keep on killing her forever. For instance, how about I drown her in a pool of sulfuric acid? If she continues to die as soon as she revives, it's as if she's not even alive in the first place, right?"
+"So here's my plan. I'll keep on killing her forever. For instance, how about I drown her in a pool of sulfuric acid? If she continues dying the very moment she revives, then that'd be the same as not being alive in the first place, right?"
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_6.txt:472]: umineko reference

File: script/Ciel/Day 13/13_01_02_CIEL13_6.txt [hunk 8]
```diff
@@ -481,26 +484,27 @@ I bought more than enough time.
...
 [sha:53c5bc86f5044523755f38f3c161120e499c7fa6]{
 -- Page 144, Offset 35654.
 -- だから、頭痛を押し殺して最後の質問だってできる。
-That's why I'm able to conceal my headache and ask my final question.
+That's why I'm able to stifle my headache and ask my final question.
 }
 [sha:80a6bf184e170e2cc45cf992c9130694afb3619a]{
 -- Page 145, Offset 35655.
 -- 「ノエル先生。これが貴女との最後の会話でしょうけど」
+// boys in the dover moment
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_6.txt:507]: image

File: script/Ciel/Day 13/13_01_02_CIEL13_6.txt [hunk 9]
```diff
@@ -636,32 +640,32 @@ Not to mention this noise.
...
 -- Page 153, Offset 35686.
 -- ノエルの指示なのか、死者たちは鉄柵の中には入ってこない。
-The Dead won't go past the fence, perhaps becasue they were instructed by Noel not to.
+The Dead aren't going past the fence, likely because Noel ordered them not to.
 }
 [sha:99a7f8708b2344e14c54634d66014c050f0ee0ed]{
 -- Page 153, Offset 35687.
 -- となれば、間髪入れずにノエル自身の追撃が来る―――！
-In that case, Noel will come after me without a moment's notice―――!
+Which means that Noel's going to come after me without a momemnt's notice―――!
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_6.txt:668]: moment's

File: script/Ciel/Day 13/13_01_02_CIEL13_6.txt [hunk 10]
```diff
@@ -671,32 +675,32 @@ In that case, Noel will come after me without a moment's notice―――!
...
 -- Page 156, Offset 35693.
 -- 「だって実際虫ケラなんだもの、<人間|あなた>たち！
-"Because in reality, you people really are like insects!
+"Because the truth is――you humans truly are like insects!
 }
 [sha:0c0d50a3592550c079eef373a246c9d9980b3940]{
 -- Page 156, Offset 35694.
 -- たった一度生きてたった一度死ぬしかないなんて、小さすぎて笑っちゃうわ！」
-To think that you people live and die only once, it's so pathetic that it makes me laugh!"
+To think that you huamns only get to live and die once! It's so pathetic that it makes me want to laugh!"
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_6.txt:703]: humans

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 13/13_01_02_CIEL13_6.txt (hunk 2 above)
Comment (nolo16) [reply script/Ciel/Day 13/13_01_02_CIEL13_6.txt:219]: image
i assumed it was something like this but yeah it is kinda stupid

**Pull request merged**